### DISCUSSION TITLE
how to keep the background remain black while the pages are colored using custom filter?

### DISCUSSION THREAD

**Discussion by uugaabuugaa** [mienaiyami/yomikiru#397] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/018a1bdf-2e3f-4ebf-871c-9f25cc6996a7)

Is there a way to make the marked background area black while the main page area have the specified color in custom filter? Darken or multiply seems to be the only blends that work for my need but they also colorize the background as well.

**Comment by mienaiyami** [comment, marked as answer]
Currently its not possible, background color is required for filters to apply correctly. While adding this feature I tried many other solutions but couldn't get good results.
Later I learned almost all other apps that support color filters have this working same way.

For some people it might just be expected to have that background color.

**Reply by uugaabuugaa** [reply]
that's a bit disappointing to hear. thanks for letting me know.
